cfgov/scripts/http_smoke_test.py

```diff
@@ -223,11 +224,11 @@ def check_urls(base, full=False):
 
     if failures:
         logger.error("These URLs failed: {}".format(failures))
-    if timeouts:
+    if len(timeouts) > ALLOWED_TIMEOUTS:
```

Comment:
Let's output the URLs that timed out even if it's okay for one to time out.

Another option is to make a note that "%d timeouts are allowed due to httpd setup time, and the following URLs timed out but are allowed".

Reply:
√
Added logging of the timeout list.

Individual timeouts were already logged (from build 1335/console:):

```
09:58:58 + python http_smoke_test.py --base http://awswebl01 -v
09:58:58 /
09:59:28 http://awswebl01/ timed out
```

Reply:
LGTM!